### PR TITLE
Add Seamus and Robert as members of this SIG

### PR DESCRIPTION
**Pull request by yJunS**

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
Sorry, I forgot to add @yJunS you as a member.

I'm wondering if you can help to add [Robert Lu](https://github.com/cdfoundation/sig-chinese-localization/pull/1#issuecomment-894806038) as a member as well.

**Comment by yJunS** [comment]
I have successfully added it locally, but there seems to be a problem with
GitHub, and I can’t push it successfully.

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Some projects from CDF already have the Chinese localization documentation. Such
 * [Shawn](https://github.com/chengjingtao) (Alauda)
 * [Suke](https://github.com/wangsrGit119) (SpringBoot Chinese Community)
 * [Seamus](https://github.com/yJunS) (Jenkins Chinese Community)
+* [Robert Lu](https://github.com/robberphex) (Robert)
```
Comment (LinuxSuRen) [inline README.md]: Robert is a personal name instead of a company name.

Comment (yJunS) [reply README.md]: Please @robberphex provide the company name

Comment (robberphex) [reply README.md]: ` * [Robert Lu](https://github.com/robberphex) (Alibaba Cloud) `

Comment (yJunS) [reply README.md]: Already modify.

File: README.md [hunk 2]
```diff
@@ -35,6 +35,8 @@ Some projects from CDF already have the Chinese localization documentation. Such
 * [Lan](https://github.com/liangyuanpeng) (Yunhorn)
 * [Shawn](https://github.com/chengjingtao) (Alauda)
 * [Suke](https://github.com/wangsrGit119) (SpringBoot Chinese Community)
+* [Seamus](https://github.com/yJunS) (Jenkins Chinese Community)
+* [Robert Lu](https://github.com/robberphex) (Alibaba Cloud)
```
Comment (LinuxSuRen) [inline README.md]: The name list should be in alphabetical order. So, please put Robert before letter S.

`A...R-S...Z`

Comment (yJunS) [reply README.md]: Done.

**Pull request merged**